gui/session/minimap/MiniMapIdleWorkerButton~boongui.js

```diff
@@ -6,5 +6,15 @@ MiniMapIdleWorkerButton.prototype.rebuild = function()
 		"excludeUnits": []
 	}).length;
 	this.idleWorkerButton.enabled = totalNumberIdleWorkers > 0;
+
+	const waitedTime = Date.now() - this.lastBeepTime;
+	if(this.idleWorkerButton.enabled && waitedTime * Math.min(totalNumberIdleWorkers, 5) > 10000)
```

Comment:
I would rather have the sound only for the woman/ support units.

`"idleClasses": ["Support"],`

and it might be better to add an option to turn it off, probably be the default.

Reply:
> I would rather have the sound only for the woman/ support units.

i disagree a bit. yes sometimes troube collecting somwhere and then the idle sound will distubing. so the idle sound maybe sound not so very loud. i my game play it produced a bit a change of behavior. tried get all doing always doning something. i played it often like this and it helps a lot. in one hand it feels a bit complicated if only some units gehts this feature. maybe best put it into the configuration.
[] only beep idle for women / support units
[] stop beep  for women / support units if greater then 150 population
[] use different idle beep sound for  women / support